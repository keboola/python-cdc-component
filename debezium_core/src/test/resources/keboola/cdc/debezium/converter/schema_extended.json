{
  "type": "struct",
  "fields": [
    {
      "type": "int32",
      "optional": false,
      "default": "0",
      "field": "id"
    },
    {
      "type": "string",
      "optional": false,
      "field": "name"
    },
    {
      "type": "string",
      "optional": true,
      "field": "description"
    },
    {
      "type": "int64",
      "optional": true,
      "name": "org.apache.kafka.connect.data.Timestamp",
      "version": 1,
      "field": "timestamp_col"
    },
    {
      "type": "string",
      "optional": true,
      "field": "kbc__operation"
    },
    {
      "type": "int64",
      "optional": true,
      "field": "kbc__event_timestamp"
    },
    {
      "type": "string",
      "optional": true,
      "field": "__deleted"
    }
  ],
  "optional": false,
  "name": "testcdc.inventory.products.Value"
}
