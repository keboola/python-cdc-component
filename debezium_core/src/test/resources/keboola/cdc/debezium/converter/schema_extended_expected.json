[
   {
      "type":"int32",
      "optional":false,
      "default":"0",
      "field":"id",
      "orderEvent":false
   },
   {
      "type":"string",
      "optional":false,
      "field":"name",
      "orderEvent":false
   },
   {
      "type":"string",
      "optional":true,
      "field":"description",
      "orderEvent":false
   },
   {
      "type":"double",
      "optional":true,
      "field":"weight",
      "orderEvent":false
   },
   {
      "type":"double",
      "optional":true,
      "field":"weight-with-dash",
      "orderEvent":false
   },
   {
      "type":"string",
      "optional":true,
      "field":"kbc__operation",
      "orderEvent":false
   },
   {
      "type":"int64",
      "optional":true,
      "field":"kbc__event_timestamp",
      "orderEvent":false
   },
   {
      "type":"string",
      "optional":true,
      "field":"__deleted",
      "orderEvent":false
   },
   {
      "type":"int",
      "optional":false,
      "version":1,
      "field":"event_order",
      "orderEvent":true
   },
   {
      "type":"int",
      "optional":false,
      "version":1,
      "field":"kbc__batch_event_order",
      "orderEvent":true
   },
   {
      "type":"int64",
      "optional":true,
      "name":"org.apache.kafka.connect.data.Timestamp",
      "version":1,
      "field":"timestamp_col",
      "orderEvent":false
   }
]
